apps/chat/message_handlers.py

```diff
@@ -19,6 +20,8 @@
 from apps.chat.exceptions import AudioSynthesizeException, MessageHandlerException
 from apps.experiments.models import ExperimentSession, SessionStatus
 
+RESET_COMMAND = "/ocs-reset"
```

Comment:
Why not just `/reset`? Might be nice to support multiple options which could make sense to different users (and different languages).

e.g. `/clean-slate`, `/startover`, `/newchat`

Reply:
> Why not just /reset?

I don't want this to interfere with "native" commands. On telegram, anything starting with / is a command, so I think prepending "ocs-" makes it a bit more specific to OCS. I don't have strong feelings on this though and can be easily persuaded

> Might be nice to support multiple options which could make sense to different users

I like this


> and different languages

Hmm this might be trickier